### PR TITLE
Remove packages routes and clean up controllers

### PR DESCRIPTION
**Pull request by saraycp**
Remove unused routes and actions for packages and tidy up the controller.

This PR depends on #8289, which should be merged before.

### CODE REVIEW

**Comment by hennevogel** [comment]
@saraycp if you attach the do-not-merge label, please explain why you do this.

**Comment by saraycp** [comment]
@hennevogel I wrote in the description:

> This PR depends on #8289, which should be merged before.

But I should have added "So I set do-not-merge label", maybe :smiley:

**Comment by saraycp** [comment]
> Everything with `render_dialog` needs to be removed. It's for sure not used in the Bootstrap UI. Some partials have the same name as those actions, but they aren't related. Those partials are the Bootstrap version of those Bento modals. This is the only relation they have.
> 
> See [my comment](https://github.com/openSUSE/open-build-service/pull/8325#issuecomment-530779012) for explanations.

@dmarcoux, there are two `render_dialog` in the package controller, but I can't removed them because the actions are not refactored. I wrote it as comments to be aware. I'm going to create some issues as well.

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -7,40 +7,27 @@ class Webui::PackageController < Webui::WebuiController
...
-                                         :requests, :statistics, :commit, :revisions, :submit_request_dialog,
-                                         :branch_diff_info,
-                                         :add_person, :add_group, :rdiff,
-                                         :save, :save_meta, :delete_dialog,
-                                         :remove, :add_file, :save_file, :remove_file, :save_person,
-                                         :save_group, :remove_role, :view_file,
-                                         :abort_build, :trigger_rebuild, :trigger_services,
-                                         :wipe_binaries, :buildresult, :rpmlint_result, :rpmlint_log, :meta,
-                                         :attributes, :edit, :files, :users, :binary_download]
+  before_action :set_project, only: [:show, :index, :users, :linking_packages, :dependency, :binary, :binaries, :requests, :statistics, :revisions,
```
Comment (dmarcoux) [inline src/api/app/controllers/webui/package_controller.rb]: `:linking_packages` needs to be removed from all filters (`before_action` and such)

Comment (krauselukas) [reply src/api/app/controllers/webui/package_controller.rb]: @dmarcoux thanks, overlooked it, but I'm currently going through the commits :)

File: src/api/app/controllers/webui/package_controller.rb [hunk 2]
```diff
@@ -241,28 +223,6 @@ def revisions
     switch_to_webui2
   end
 
-  # TODO: bento_only
-  def submit_request_dialog
```
Comment (dmarcoux) [inline src/api/app/controllers/webui/package_controller.rb]: There are still specs for this action in [spec/controllers/webui/package_controller_spec.rb](https://github.com/openSUSE/open-build-service/blob/69274b51f0b11298cdd7db49852fd14e1f769622/src/api/spec/controllers/webui/package_controller_spec.rb#L1360)

File: src/api/app/controllers/webui/package_controller.rb (hunk 1 above)
Comment (krauselukas) [reply src/api/app/controllers/webui/package_controller.rb]: @dmarcoux removed it

File: src/api/app/controllers/webui/package_controller.rb (hunk 2 above)
Comment (krauselukas) [reply src/api/app/controllers/webui/package_controller.rb]: @dmarcoux thanks and done :+1:

**Pull request merged**